aflr/api_resources/file.py

```diff
@@ -1,19 +1,60 @@
+from posixpath import basename
 import aflr
-from aflr.api_request import APIRequest
+from aflr.helper_classes import ListableResource
+from tkinter import Tk
+from tkinter.filedialog import askopenfilename
+import os
+import requests
+import json
 
 
-class File(APIRequest):
+class File(ListableResource):
     OBJECT_NAME = "file"
+    resource_path = "/file"
+    custom_audio_upload = "/file/customaudio/uploadurl?filename="
```

Comment:
i'd have only `custom_audio_resource_path = "file/customaudio"` and leave the rest to the methods

Reply:
done